### DISCUSSION TITLE
Puedo subir una cata con error?

### DISCUSSION THREAD

**Discussion by NayGL** [LaunchX-InnovaccionVirtual/CursoIntroPython#304] [Q&A] [answered]
Hola, de casualidad saben si puedo modificar una Kata después de subirla y enviar el enlace? Qué pasa si una de mis katas tiene errores que me han sido difíciles solucionar, aun así la puedo subir? Muchas Gracias!!

**Comment by JavierMendez-Coder** [comment, marked as answer]
Buenas noches. Contesto tus preguntas en orden:
1. Sí puedes modificar las katas una vez subidas. Simplemente arrastra los nuevos archivos, y GitHub debería encargarse de reemplazarlos automáticamente, o en caso de que uses GitBash o una terminal simplemente haces `push` del repositorio completo. 
2. A la hora agregar/elimanar/modificar archivos no es necesario preocuparte por el enlace, ya que, este no cambia independiemente de lo que hagas en tu repositorio.
3. Si mal no me equivoco, Fernanda comentó que no hay problema siempre y cuando lo intentes.

Respecto a esto último, puedes comparar tu código con el que aparece en la parte inferior de los katas, ya que ahí te brindan las soluciones a los ejercicios.
Saludos, 👋

**Reply by NayGL** [reply]
Gracias!!
